test/e2e/txmgr/txSender.go

```diff
@@ -0,0 +1,146 @@
+package txmgr
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+	"strings"
+
+	"github.com/omni-network/omni/contracts/bindings"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+	"github.com/omni-network/omni/lib/netconf"
+	"github.com/omni-network/omni/lib/txmgr"
+	t "github.com/omni-network/omni/lib/txmgr"
```

Comment:
best not use weird aliases like this it isn't very readable. I had not idea this was a package when I was reading the code, it looked like some locally variable...